exo.js

```diff
@@ -0,0 +1,80 @@
+//------ Exercice 1 ------
```

Comment:
Please put each exercice inside a function and return the final value instead of log it.
Please use return instead of console.log so that we can easily resuse the function and test is